app/ts-store/transport/handler/handlers_process.go

```diff
@@ -117,6 +119,50 @@ func (h *ShowTagValuesCardinality) Process() (codec.BinaryCodec, error) {
 	return h.rsp, nil
 }
 
+func (h *ShowQueries) Process() (codec.BinaryCodec, error) {
+	var queries []*netstorage.QueryExeInfo
+
+	// get all query exe info from all query managers
+	getAllQueries := func(manager *query.Manager) {
+		queries = append(queries, manager.GetAll()...)
+	}
+	query.VisitManagers(getAllQueries)
+	rsp := &netstorage.ShowQueriesResponse{}
+	rsp.QueryExeInfos = queries
+
+	return rsp, nil
+
+}
+
+func (h *KillQuery) Process() (codec.BinaryCodec, error) {
+	qid := h.req.GetQueryID()
+	isExist := false
+	isAborted := false
+
+	killQueryByIDFn := func(manager *query.Manager) {
+		// qid is not in current manager, or it has been aborted successfully
+		if manager.Get(qid) == nil || isAborted {
+			return
+		} else {
+			isExist = true
+			manager.Abort(qid)
+			isAborted = manager.Aborted(qid)
+		}
+	}
+	query.VisitManagers(killQueryByIDFn)
+	err := &errno.Error{}
+	if !isExist {
+		err = err.SetErrno(errno.ErrQueryNotFound)
+		err.SetMessage(fmt.Sprintf("the query %d is not running on this store node", qid))
```

Comment:
1. define error code in `lib/errno/code.go`
2. define error message in `lib/errno/message.go`
3. e.g. `errno.NewError(errno.ErrQueryNotFound, qid)`